tomviz/python/LabelObjectAttributes.py

```diff
@@ -26,8 +26,7 @@ def transform_scalars(self, dataset):
 
         returnValues = None
 
-        scalarType = dataset.GetScalarType()
-        if scalarType == VTK_FLOAT or scalarType == VTK_DOUBLE:
```

Comment:
Here is an example of a change in the way we were checking to see if the type was a float. Instead of checking the vtk type, we check the numpy type with `np.issubdtype(dtype, np.floating)`.